prisma-data-platform/pulse/starter/global.d.ts

```diff
@@ -0,0 +1,6 @@
+// global.d.ts
+declare namespace NodeJS {
+	export interface ProcessEnv {
```

Comment:
tabs here, let's standardize on spaces.

Reply:
I updated the format using prettier. And added a `.vscode` config file.